README.md

```diff
@@ -1,16 +1,19 @@
-# Ed25519Signature2020 Interoperability Test Suite
+# di-ed25519signature2020-test-suite
+
+> Test Suite for [Ed25519Signature2020](https://w3c-ccg.github.io/lds-ed25519-2020/).
```

Comment:
URL may still need to be changed.
```suggestion
# Test Suite for [Ed25519Signature2020](https://w3c-ccg.github.io/lds-ed25519-2020/)
```

Reply:
@msporny Thanks, fixed the URL [71777d5](https://github.com/w3c-ccg/di-ed25519signature2020-test-suite/pull/39/commits/71777d5dea603ded1cba9f021bfd20296fbfa19d)